### PR TITLE
[16.0][MIG] account_invoice_discount_display_amount: Migration to 16.0

### PR DESCRIPTION
**Pull request by samoros1**

### CODE REVIEW

**Comment by samoros1** [comment]
Module migrated to version 16.0

cc https://github.com/APSL 148568

@miquelalzanillas @lbarry-apsl @javierobcn @peluko00 please review

Issue https://github.com/OCA/account-invoicing/issues/1250

**Review by rafaelbn — APPROVED** [review]
👍🏼 

imagen

**Review by miquelalzanillas — APPROVED** [review]
Functional ok!

**Review by peluko00 — APPROVED** [review]
Functional review ok!

**Review by AungKoKoLin1997 — APPROVED** [review]
LGTM

**Review by aliciagaarzo — APPROVED** [review]

**Comment by SoniaViciana** [comment]
This PR has 5 reviewers and approvals, can it be merged?

Thanks,

@pedrobaeza

File: account_invoice_discount_display_amount/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+* Manuel Regidor <manuel.regidor@sygel.es>
+* Alexei Rivera <arivera@archeti.com>
+- [APSL](https://apsl.tech):
+  - Santi Amorós \<<samoros@apsl.net>\>
```
Comment (pedrobaeza) [inline account_invoice_discount_display_amount/readme/CONTRIBUTORS.rst]: Incorrect RST syntax. This is RST, not Markdown (you can convert all to MD, but not mix them, and more having .rst extension).

**Review by pedrobaeza — DISMISSED** [review]

**Comment by SoniaViciana** [comment]
could you review to be able to merge @samoros1 ?
Thanks!

File: account_invoice_discount_display_amount/readme/CONTRIBUTORS.rst (hunk 1 above)
Comment (peluko00) [reply account_invoice_discount_display_amount/readme/CONTRIBUTORS.rst]: Fix it!

**Comment by peluko00** [comment]
> could you review to be able to merge @samoros1 ? Thanks!

Done!

**Comment by aritzolea** [comment]
@pedrobaeza Could it be merged after the correction on Readme? Thanks

**Pull request merged**